src/deploy/osp_deployer/settings/sample_csp_profile.ini

```diff
@@ -171,6 +173,19 @@ StorageBond1Interface1=em2
 StorageBond1Interface2=p2p2
 StorageBondInterfaceOptions=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 
+# Note: the following interfaces need to be changed 
+# as per the server model
+# R630 should use 'p2p1', 'p3p1', 'p2p2', 'p3p2'
+# R730 should use 'p4p1', 'p5p1', 'p4p2', 'p5p2'
+# The following lines must be uncommented only if OVS-DPDK is set to enable 
+# See Dell NFV Settings section -> option ovs_dpdk_enable
+ComputeOvsDpdkInterface1=p2p1
+ComputeOvsDpdkInterface2=p3p1
+ComputeOvsDpdkInterface3=p2p2
+ComputeOvsDpdkInterface4=p3p2
+BondInterfaceOvsOptions=bond_mode=balance-tcp lacp=active
+
```

Comment:
i m a little confused by the above nics, our default nics enumeration for computes is em1/p1p1 em2/p1p2. This is the default defined in both our sample and nic config files, where do those p2p* & p4p* & p5p* come from ?

Reply:
Those bond are used only for Dpdk, remember the hardcoding in the code? regular bonding on compute remain the same aka em1/p1p1 em2/p1p2.